channels-rke2.yaml

```diff
@@ -88,7 +88,7 @@ releases:
         type: array
       kube-proxy-arg:
         type: array
-    charts:
+    charts: &charts
```

Comment:
Should this have a `-v1` like the other ones? 

Usually I wouldn't expect the charts to be the same from each RKE2 release, but understand that this use case is unique cause of how close they are to each other.

Reply:
We should decide if we need to even use this syntax when referencing charts cause I'd imagine them changing between releases and we should explicitly change them between releases. 

Note: This wouldn't need to block approving this PR, but just a comment for the next PR.

Reply:
I think in this case it should be fine but am happy to put the `v1` on the end of the charts section anchor.

Reply:
spoke with @ibuildthecloud and he said we should add the `v1` to charts, but since we're going to consolidate to only 1 version by the time we ship, we will merge as is for now.